### ISSUE TITLE
[8.3.0] Unable to exempt authorized people from crafting fee

### ISSUE THREAD

**Issue by Praefator** [StrangeLoopGames/EcoIssues#13051]
See included screenshots. Previously, authorized users were exempt from paying a crafting fee if one was set - it would include a message to the tune of "fee of xx currency waived for authorized user". One of my players mentioned having to pay a fee regardless, which prompted an investigation and subsequently this report. 

![20190904000115_1](https://user-images.githubusercontent.com/841990/64211828-b8650800-cea7-11e9-8162-68d89241bb37.jpg)
![20190904000120_1](https://user-images.githubusercontent.com/841990/64211829-b8650800-cea7-11e9-8081-1ac891cc7914.jpg)

**Comment by SlayksWood** [comment]
Can't reproduce. 
If I autorized somebody to my territory which have public craft station then she doesn't pay tax for using my craft table. The message "fee of xx currency waived for authorized user" apperes but dissappears very fast. 
![image](https://user-images.githubusercontent.com/45708377/64236926-07d92180-cf04-11e9-92fd-491ee4e1c111.png)
We checked the debit from bank accounts. Everything works well. If you are authorized on the territory, then money is not taken from you.
Can you provide a little more information?

**Comment by Praefator** [comment]
I think the problem lies in the way property authorization was tested. As you can see in my screenshot, I was authorized **on the blast furnace object**. You're stating **territory** authorization. That's the whole deed, which given a store on the same deed is way too blunt of a method to be usable. 

We would like to be able to authorize specific players on specific workspaces for services rendered in making the station possible, while using the 'craft per minute' system to rent out the property, recoup investment and make leapfrogging for late joiners easier. 

In the current (bugged?) state, I would only be able to authorize that person on the deed (thus removing the ability to use my store) and not allow fine grained control over who pays for which usage.

**Comment by SlayksWood** [comment]
@Praefator 
You can place your store at another deed. And another player could use your shop.
If you add public craft station with payment then people which you autorized it's people who can use this station with payment. I think It's intented behavior.
It's more look like Suggestion, because from another side I can want to share my public craft station for little count of people or for only one.

**Comment by Praefator** [comment]
Yes, I **could** place my store somewhere else. However I am playing in a highly collaborative server, where space in a well-organized capital is at a premium. I cannot just pick up and move an entire mechanics and industry/electronics workshop elsewhere because the game currently seems to lack the functionality to allow for a precise amount of control over the way we organize our collaborative efforts between an engineer and a smith on a microscopic scale and economic activities on a macroscopic level. 

The point I'm arguing for, is to increase the quality of life of us highly organized, highly collaborative players. Seeing as how the goal for 9.0 has been stated to **increase player agency**, allowing people to jump right in and participate at **any** point of a map's lifespan, this issue is more of a 'feature' than a mere 'suggestion'. 

To put things into context: in my experience on the Eco Discord (explicitly stating as such), as well as months of hosting a server, people don't tend to join past the first week, due to the notion that they will be too far behind everyone else and thus either miss out or spend the remainder of the map 'catching up'. That's not a pleasant experience, neither for the player nor for us as server owners. 

Therefore, rather than confronting people who want to take up engineering with a huge upfront cost in both building materials and the iron they'll need to get a tier 2 mechanics room room up just to get started (whereas it took me about 460k in currency, versus a cost per unit of 40 per log, or >15k logs), I seek to do the opposite. I offer them a chance to rent my machines at a fair rate (with a ROI of 30 continuous hours), meaning they need far less of a reserve to get started. 

Couple that with the ability for people to choose their own spawn point (which I heard was planned for 9.0?), and people will have a greatly increased **quality of life**, as well as a large increase in **player agency**, which should be the goal here. 

TLDR: I disagree with the notion of this being a 'suggestion' and would rather see it be a 'feature' to be considered for increasing QOL and player agency in line with the stated goal of having increased player agency.
